sdk/rust/oak/build.rs

```diff
@@ -15,11 +15,21 @@
 //
 
 fn main() {
-    oak_utils::compile_protos(
+    oak_utils::compile_protos_internal_with_service_generator(
         &[
             "../../../oak/proto/storage_service.proto",
             "../../../oak/proto/roughtime_service.proto",
+            "../../../oak/proto/handle.proto",
         ],
         &["../../.."],
     );
+
+    let mut handle_tests_out = std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap());
```

Comment:
Is there any way to `cfg(test)` protect this, or is there just one `build.rs` invocation regardless?

Reply:
I don't think this is currently supported, see https://github.com/rust-lang/cargo/issues/1581 and https://github.com/rust-lang/cargo/issues/2549. Both issues are quite old but I couldn't find more recent discussions.